src/components/TabSelect/_TabButton.js

```diff
@@ -3,6 +3,14 @@ import styled from 'styled-components';
 
 const TabName = styled.span``;
 
+const getBgColor = (selected, secondary, theme) => {
+  if (secondary) {
+    return theme.colors.sand;
```

Comment:
Any reason why we'd prefer a fixed color here instead of `'transparent'`?

Reply:
Great point and kinda what I was alluding to in the description regarding background color. We have to set the color explicitly for the tab button itself otherwise the top border of the content shows.

Reply:
We could add a prop for `secondaryBg` if there is a need to override `theme.colors.sand` but I can't see any way to use transparent unfortunately

Reply:
Ok I see what you mean now and I agree we should only try to address this if the use case occurs. :)